nginx/hellogov.conf

```diff
@@ -0,0 +1,19 @@
+server {
+  listen 80;
+  listen [::]:80;
+  server_name hellogov.org;
```

Comment:
I've been putting a placeholder (###DOMAIN_NAME##) in this file and doing a find-replace on it with a domain name the box will serve from. I'll point it out in my next commit. Its not the most robust method, but it lets us deploy to multiple domain names, which is something I've been doing often.